telegram/_message.py

```diff
@@ -359,7 +359,7 @@ class Message(MaybeInaccessibleMessage):
             about the animation. For backward compatibility, when this field is set, the document
             field will also be set.
         game (:class:`telegram.Game`, optional): Message is a game, information about the game.
-            `More about games >> <https://core.telegram.org/bots/api#games>`_.
+            `More about games >> <https://docs.python-telegram-bot.org/telegram.games-tree.html>`_.
```

Comment:
These should also work with sphinxs reference system. I guess <code>:ref:\`More about games >> \<games\>\`</code> should work, or `:doc:`. If you can not get the link to point to the correct location, you might need to introduce a refrence target manually - I'm sure that your favorite search engine will help you with that :)